src/main/java/cascading/tap/hcatalog/HCatTap.java

```diff
@@ -0,0 +1,173 @@
+/*
+* Copyright (c) 2007-2016 Concurrent, Inc. All Rights Reserved.
+*
+* Project and contact information: http://www.cascading.org/
+*
+* This file is part of the Cascading project.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package cascading.tap.hcatalog;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.mapred.OutputCollector;
+import org.apache.hadoop.mapred.RecordReader;
+import org.apache.hive.hcatalog.mapreduce.HCatInputFormat;
+import org.apache.hive.hcatalog.mapreduce.HCatOutputFormat;
+import org.apache.hive.hcatalog.mapreduce.OutputJobInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import cascading.flow.FlowProcess;
+import cascading.scheme.hcatalog.HCatScheme;
+import cascading.tap.Tap;
+import cascading.tap.hadoop.io.HadoopTupleEntrySchemeCollector;
+import cascading.tap.hadoop.io.HadoopTupleEntrySchemeIterator;
+import cascading.tuple.TupleEntryCollector;
+import cascading.tuple.TupleEntryIterator;
+
+import com.twitter.elephantbird.mapred.input.DeprecatedInputFormatWrapper;
+import com.twitter.elephantbird.mapred.output.DeprecatedOutputFormatWrapper;
+
```

Comment:
Again, I am not the biggest fan of dragging in yet another library. How big are these things? Can we maybe just copy the code over? (with attribution of course)